spendingPlansConfig.json.example

```diff
@@ -0,0 +1,24 @@
+[
```

Comment:
qq: I understand this is an example file, but why use `.json.example` instead of `.example.json` so it would allow for syntax highlighting and better tooling support?

Reply:
Valid point, changed it to `spendingPlansConfig.example.json`